models/maker/ethereum/maker_ethereum_accounting.sql

```diff
@@ -924,256 +934,479 @@ WITH dao_wallet AS (
+            , 'Sin Inflows' AS descriptor
+            , CAST(NULL as string) AS ilk
+        FROM sin_inflows
+
+        UNION ALL
+
+        SELECT ts
+            , hash
+            , code
+            , value
+            , 'DAI' AS token
+            , 'DSR Flows' AS descriptor
+            , CAST(NULL as string) AS ilk
+        FROM dsr_flows
+
+        UNION ALL
+
+        SELECT ts
+            , hash
+            , code
+            , value
+            , token
+            , 'Treasury Flows' AS descriptor
+            , CAST(NULL as string) AS ilk
+        FROM treasury_flows
+
+        UNION ALL
+
+        SELECT ts
+            , hash
+            , code
+            , value
+            , 'DAI' AS token
+            , 'Loan Draws/Repays' AS descriptor
+            , ilk
+        FROM loan_actions
+
+        UNION ALL
+
+        SELECT ts
+            , hash
+            , code
+            , value
+            , 'DAI' AS token
+            , 'D3M Revenues' AS descriptor
+            , ilk
+        FROM d3m_revenues
+
+        UNION ALL
+
+        SELECT ts
+            , hash
+            , code
+            , value
+            , 'DAI' AS token
+            , 'PSM Yield' AS descriptor
+            , ilk
+        FROM psm_yield
+
+        UNION ALL
+
+        SELECT ts
+            , hash
+            , code
+            , value
+            , 'MKR' AS token
+            , 'MKR Vest Creates/Yanks' AS descriptor
+            , NULL AS ilk
+        FROM mkr_vest_creates_yanks
+
+        UNION ALL
+
+        SELECT ts
+            , hash
+            , code
+            , value
+            , 'MKR' AS token
+            , 'MKR Pause Proxy Trxns' AS descriptor
+            , NULL AS ilk
+        FROM pause_proxy_mkr_trxns
+
+        UNION ALL
+
+        SELECT ts
+            , hash
+            , code
+            , value
+            , token
+            , 'Accounting Plugs' AS descriptor
+            , ilk
+        FROM hashless_trxns
+
+
+        UNION ALL
+
+        SELECT ts
+            , 'noHash:dailyMarkToMarket' AS hash
+            , 19999 AS code
+            , 0 AS value
+            , token
+            , 'Currency Translation to Presentation Token' AS descriptor
+            , NULL AS ilk
+        FROM m2m_levels
+
+        UNION ALL
+
+        SELECT ts
+            , 'noHash:dailyMarkToMarket' AS hash
+            , 29999 AS code
+            , 0 AS value
+            , token
+            , 'Currency Translation to Presentation Token' AS descriptor
+            , NULL AS ilk
+        FROM m2m_levels
+
+        UNION ALL
+
+        SELECT ts
+            , 'noHash:dailyMarkToMarket' AS hash
+            , 39999 AS code
+            , 0 AS value
+            , token
+            , 'Currency Translation to Presentation Token' AS descriptor
+            , NULL AS ilk
+        FROM m2m_levels
+    ) unioned
+    USING (code)
+    LEFT JOIN token_prices
+    ON unioned.ts::DATE = token_prices.ts::DATE
```

Comment:
we are migrating away from `::` usage for datatypes, please use `cast(column as type)` instead through the script(s)